api/peer_connection_interface.h

```diff
@@ -607,7 +607,8 @@ class RTC_EXPORT PeerConnectionInterface : public rtc::RefCountInterface {
     // The min time period for which a candidate pair must wait for response to
     // connectivity checks it becomes inactive. This parameter overrides the
     // default value in the ICE implementation if set.
-    absl::optional<int> ice_inactive_timeout;
+    // RingRTC change to configure inactive timeout
+    absl::optional<int> ice_inactive_timeout = 30 * 1000; // 30 seconds
```

Comment:
"This paramater overrides the default value in the ICE implementation" sounds like there's a default value somewhere else we could change, rather than forcing this setting here. I guess it might be in a third_party module though?

Reply:
The default value is [here](https://github.com/signalapp/webrtc/blob/f17c45862d69bf98fc64da1bbbd56e2b7579148e/p2p/base/p2p_constants.cc#L70), so changing that instead is an option. I set this value though since we're configuring PlanB here, so we'll already be looking at this file when merging upstream.